src/components/ProjectCard.jsx

```diff
@@ -1,38 +1,37 @@
 import React from "react"
 
 function ProjectCard({ gh, link, title, description, tech }) {
-  return (
-    <div className="flex flex-col border shadow-sm rounded-xl p-4 md:p-5 dark:bg-gray-800 dark:border-gray-700 dark:shadow-slate-700/[.7] transition-all duration-200  hover:scale-105  cursor-pointer">
-      <div className="flex flex-col justify-between h-full">
-        <div>
-          <div className="flex justify-between">
-            <h3 className="text-lg font-bold text-white-800 dark:text-white">{title}</h3>
+  const maxLength_title = 25
+  const maxLength_techtag = 6
+
+  let newTitle = title.length > maxLength_title ? title.slice(0, maxLength_title) + "..." : title
```

Comment:
we can use `line clamp` too.

Reply:
just add `line-clamp-2` in the line where you want to display. I already added it to the description. 
- It will show `...` after 2 lines.

![image](https://user-images.githubusercontent.com/88102392/231085605-af7157ed-116c-4f1f-a1ce-e7295cef1ae8.png)


```
<p className="mb-3 line-clamp-2">{description}</p>
```

Reply:
When you will add `pagination` then change this